set.mm

```diff
@@ -748683,6 +748793,185 @@ set of all the singletons of (proper) ordered pairs over the elements of
       VN $.
   $}
 
+  ${
+    fcores.f $e |- ( ph -> F : A --> B ) $.
+    fcores.e $e |- E = ( ran F i^i dom G ) $.
+    fcores.p $e |- P = ( `' F " E ) $.
+    $( Lemma 1 for ~ fcores .  (Contributed by AV, 17-Sep-2024.) $)
+    fcoreslem1 $p |- ( ph -> P = ( `' F " dom G ) ) $=
+      ( ccnv crn cdm cin cima imaeq2i eqtri wfun wceq ffund cnvimainrn syl5eq
+      syl ) ADFKZFLGMZNZOZUDUEOZDUDEOUGJEUFUDIPQAFRUGUHSABCFHTUEFUAUCUB $.
+
+    fcores.y $e |- Y = ( F |` P ) $.
```

Comment:
It would be clearer if you rename X to Y and Y to X everywhere.

Reply:
You are right: th current Y corresponds to F, and the current X corresponds to G.  I will add a TODO here and will fix this in a subsequent PR.

Reply:
Done already in this PR.